liquidctl/driver/hwmon.py

```diff
@@ -35,6 +35,9 @@ def get_string(self, name):
     def get_int(self, name):
         return int(self.get_string(name))
 
+    def write_str(self, name, value):
+        (self.path / name).write_text(value)
```

Comment:
For symmetry, I think a `write_int` API would be better, and move the `str()` conversion within it.

On a related note, it would also probably make sense to rename `get_int` to `read_int`, also for symmetry.

Reply:
Since renaming `get_int` to `read_int` touches more drivers than `aquacomputer.py`, do you want me to create a separate PR right after this one or do it here?

Reply:
You're right, that would be better.

Reply:
Will do!